app/compare/diff_chunk.tsx

```diff
@@ -59,3 +60,17 @@ export default class DiffChunk extends React.Component<DiffChunkProps, DiffChunk
     return nodes;
   }
 }
+
+function LineContent({ line }: { line: string }) {
```

Comment:
all of this and ``/__HIDDEN__${Math.random()}/`` for slightly lighter text color 😂